Documentation/dev-internal/discovery_protocol.md

```diff
@@ -109,7 +109,7 @@ The status for this discovery token, including the machines that have been regis
 
 The repository is located at https://github.com/coreos/discovery.etcd.io. It could be used to build a custom discovery service.
 
-[api]: ../v2/api.md#waiting-for-a-change
-[cluster-size]: ../v2/admin_guide.md#optimal-cluster-size
+[api]: /docs/v2/api#waiting-for-a-change
```

Comment:
this and next one (cluster-size) links are broker - is it because of restructuring here and will be fixed once this PR merged? Thanks!